junit5/src/test/resources/test-service-https.json

```diff
@@ -59,7 +59,7 @@
     }
   },
   "meta": {
-    "schemaVersion": "v1",
+    "schemaVersion": "v2",
```

Comment:
This simulation is actually in a v1 schema, changing it to v2 is a bit misleading. Is it possible not to drop v1 support yet while reading the version from the simulation config? The reason it was hardcoded is because Hoverfly automatically upgrades simulation to the latest schema.

Reply:
The reason why I dropped the v1 support is that v1 is using different types of the parameters than is configured in hoverfly-java. 
But I can automatically change the version from v1 to v2 - I'll send commit in a minute.

> because Hoverfly automatically upgrades simulation to the latest schema

so why there is the logic for unmarshalling the JSON content to the relative simulation view? https://github.com/SpectoLabs/hoverfly/blob/master/core/handlers/v2/simulation_views.go#L35-L85
From that code, I got the feeling that hoverfly distinguishes which schema version is specified, but I could have missed something ...

Reply:
Hoverfly is backward compatible with all the old versions of simulation files, that's why there is logic to unmarshal each version and does some transformation to upgrade. 

hoverfly-Java handles all version as well, but we decided not to keep separate set of Simulation models, instead it uses custom serialiser to convert v1 to have field matches. But export will only output the latest version that's why the version number was hard coded. We try not to Impact the users when we do an scheme upgrade.